samples/CMakeLists.txt

```diff
@@ -104,53 +104,69 @@ endif()
 # Automatically include the multiprotocol_rpmsg as child image when both RPMSG HCI
 # and RPMSG 802.15.4 Serialization are enabled or the hci_rpmsg sample when
 # only RPMSG HCI is enabled. For any of these samples change the board to be the network core.
-if (CONFIG_BT_RPMSG_NRF53)
-  if (CONFIG_SOC_NRF5340_CPUAPP)
-
-    if (CONFIG_NRF_802154_SER_HOST)
-      set(NETCORE_IMAGE "multiprotocol_rpmsg")
-      set(NETCORE_IMAGE_PATH "${NRF_DIR}/samples/nrf5340/${NETCORE_IMAGE}")
-      message("Adding 'multiprotocol_rpmsg' firmware as child image since "
-      "CONFIG_BT_RPMSG_NRF53 and CONFIG_NRF_802154_SER_HOST are set to 'y'")
+if (CONFIG_NCS_AUTO_INCLUDE_NET_CORE_IMAGE)
+  if (CONFIG_BT_RPMSG_NRF53)
+    if (CONFIG_SOC_NRF5340_CPUAPP)
```

Comment:
We don't need any of these any more since `CONFIG_NCS_AUTO_INCLUDE_NET_CORE_IMAGE` depends on `NRF_SOC_NRF5340_CPUAPP`, this also means that we can remove the error messages below for not supporting the child image for non 53 boards.

Reply:
I was also thinking of having only one add_child_image call, would that be ok ? I guess this also goes for the network_core_firmware_update_enable(), although I'm not sure why it's not being called for the 802154_rpmsg case. What do you think ?

Reply:
See https://github.com/nrfconnect/sdk-nrf/pull/5052

Reply:
See https://github.com/nrfconnect/sdk-nrf/pull/5052

Reply:
Yes, that's better thanks :)